org.telegram.desktop.json

```diff
@@ -38,6 +38,14 @@
     "cleanup-commands": [
         "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
     ],
+    "build-options": {
```

Comment:
Please set this per-module where needed instead of globally.

Reply:
No, this is a bad idea. It is better to have all tdesktop dependencies have without assertions to avoid crashes.